{
  "_annotation": {
    "cna": "github_m",
    "cve_id": "CVE-2024-34355",
    "generated_from": "https://raw.githubusercontent.com/anchore/cve-data-enrichment/main/data/anchore/2024/CVE-2024-34355.json",
    "reason": "Added CPE configurations because not yet analyzed by NVD.",
    "references": [
      "https://github.com/TYPO3/typo3/commit/56afa304ba8b5ad302e15df5def71bcc8d820375",
      "https://typo3.org/security/advisory/typo3-core-sa-2024-007",
      "https://github.com/TYPO3/typo3/security/advisories/GHSA-xjwx-78x7-q6jc"
    ]
  },
  "cve": {
    "configurations": [
      {
        "nodes": [
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:typo3:typo3:*:*:*:*:*:php:*:*",
                "matchCriteriaId": "A408A24A-14A3-5608-924B-04BFECF1C86B",
                "versionEndExcluding": "13.1.1",
                "versionStartIncluding": "13.0.0",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          }
        ]
      }
    ]
  }
}